cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/orion/OrionBackend.java

```diff
@@ -17,27 +17,37 @@
  */
```

Comment:
The corresponding entry in the CHANGES_NEXT_RELEASE file (mentioning issue #1477) should be added.